src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -314,10 +356,18 @@ private string[] GenerateTableRow(string[] values, int[] alignment, DisplayCells
             for (int row = 0; row < rows.Length; row++)
             {
                 StringBuilder sb = new StringBuilder();
-                // for a give row, walk the columns
+                // for a given row, walk the columns
                 for (int col = 0; col < scArray.Length; col++)
```

Comment:
Not for th PR - we have several similar repetitive cycles which we could replace one.

Reply:
Please open an issue and mark as code cleanup.  Thanks!

Reply:
Done #6526.